CBLClient/Apps/CBLTestServer-Dotnet/TestServer/P2PMethods.cs

```diff
@@ -330,7 +330,7 @@ public class P2PMethods
                 String maxRetryWaitTime = postBody["max_timeout"].ToString();
                 if (String.IsNullOrEmpty(maxRetryWaitTime.Trim()))
```

Comment:
While I saw this, I got curious whether this is negative checking?

if input = null / empty its entering the condition, is that what needed here? or somewhere logic is flipped?